### PR TITLE
Kumarabd/feature/labs

### PR DESCRIPTION
**Pull request by kumarabd**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,16 +4,15 @@ WORKDIR /github.com/layer5io/meshery-istio
 ADD . .
 RUN GOPROXY=direct GOSUMDB=off go build -ldflags="-w -s" -a -o /meshery-istio .
 RUN find . -name "*.go" -type f -delete; mv istio /
-RUN wget -O /istio.tar.gz https://github.com/istio/istio/releases/download/1.7.3/istio-1.7.3-linux-amd64.tar.gz
+RUN wget -O /istio.tar.gz https://github.com/istio/istio/releases/download/1.5.1/istio-1.5.1-linux.tar.gz
```
Comment (leecalcote) [inline Dockerfile:7]: @kumarabd are you intentionally bundling this older version?

Comment (kumarabd) [reply Dockerfile:7]: Yes lee, this will have no effect on the latest one, but will enable us to install older versions

**Comment by kumarabd** [comment]
Also @leecalcote  let's not make a release after merging, we can run it through once and then do it.

**Comment by leecalcote** [comment]
Output from test of Enable Prometheus Dashboard:

```
DEBU[1526] received contextName: docker-desktop         
INFO[1526] Retrieved Resource of type:  and name: default 
DEBU[1526] retrieved namespace: &unstructured.Unstructured{Object:map[string]interface {}{"apiVersion":"v1", "kind":"Namespace", "metadata":map[string]interface {}{"annotations":map[string]interface {}{"linkerd.io/inject":"enabled"}, "creationTimestamp":"2020-10-09T18:08:56Z", "labels":map[string]interface {}{"istio-injection":"enabled"}, "managedFields":[]interface {}{map[string]interface {}{"apiVersion":"v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:status":map[string]interface {}{"f:phase":map[string]interface {}{}}}, "manager":"kube-apiserver", "operation":"Update", "time":"2020-10-09T18:08:56Z"}, map[string]interface {}{"apiVersion":"v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:metadata":map[string]interface {}{"f:labels":map[string]interface {}{".":map[string]interface {}{}, "f:istio-injection":map[string]interface {}{}}}}, "manager":"kubectl", "operation":"Update", "time":"2020-10-10T14:39:42Z"}, map[string]interface {}{"apiVersion":"v1", "fieldsType":"FieldsV1", "fieldsV1":map[string]interface {}{"f:metadata":map[string]interface {}{"f:annotations":map[string]interface {}{".":map[string]interface {}{}, "f:linkerd.io/inject":map[string]interface {}{}}}}, "manager":"meshery-linkerd", "operation":"Update", "time":"2020-10-10T20:54:01Z"}}, "name":"default", "resourceVersion":"128892", "selfLink":"/api/v1/namespaces/default", "uid":"a267a06e-98f0-498b-b2b5-127602ca3877"}, "spec":map[string]interface {}{"finalizers":[]interface {}{"kubernetes"}}, "status":map[string]interface {}{"phase":"Active"}}} 
INFO[1526] Updated Resource of type: Namespace and name: default 
ERRO[1526] unable to parse template: read istio/config_templates: is a directory 
```

**Review by leecalcote — APPROVED** [review]

**Pull request merged**